src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -116,7 +114,7 @@ public byte[] GetHashAndReset()
                 _hash.Initialize();
             }
 
-            _hash.TransformFinalBlock(s_empty, 0, 0);
+            _hash.TransformFinalBlock(Array.Empty<byte>(), 0, 0);
```

Comment:
> which you created @stephentoub :)

Yes :smile: I know we have this restriction on the framework components in desktop.  I was really hoping we didn't have it on some subset of corefx code, especially code in file named ".net46.cs" :)